website/docs/input.md

```diff
@@ -0,0 +1,62 @@
+---
+id: input
+title: Admission Review Input
+---
+
+The data that's passed to Gatekeeper for review follows a structure that contains the object being created, and in the case of update operations the old object being updated. You can see an example of the request structure below.
```

Comment:
Thanks for the PR!
We currently have a section in the doc for input, which points to the official k8s api docs for admissionReview. https://open-policy-agent.github.io/gatekeeper/website/docs/howto#input-review

Can you please take a look and see if that section can be improved or more easily discoverable so we don’t duplicate this information? Thanks!

Reply:
Oh I'm sorry I missed that! Thanks for pointing it out. I'll revise this

Reply:
I moved the info in how-to to the dedicated page. I think it's something that folks end up referencing, so it may be good in a dedicated page. Let me know if that sounds ok to you :)